<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 236, 239);</string>
  </property>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>640</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget1 {background-color: rgb(147, 205, 255);};

</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 239, 115);
color: rgb(0, 0, 0);
font: 75 40pt &quot;Product Sans&quot;;
font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>Modified RC4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton {background-color: rgb(255, 239, 115);
font: 500 17pt &quot;Product Sans&quot;;
color: rgb(0, 0, 0);
border-radius: 10px;}
</string>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_2 {background-color: rgb(255, 239, 115);
font: 500 17pt &quot;Product Sans&quot;;
color: rgb(0, 0, 0);
border-radius: 10px;}
</string>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
